bitrise.yml

```diff
@@ -764,7 +764,8 @@ workflows:
             set -e
             # debug log
             set -x
-            fastlane -v
```

Comment:
Could you please temporarly modify the trigger scheme so that L10nBuild workflow runs to be sure this is going to work? 
Thanks

Reply:
Testing just `L10nBuild` on Bitrise. ✅ 🎉 
https://app.bitrise.io/build/ea5fe327-965d-45f7-83ad-a30b8411e2aa

Reply:
`fastlane` has been installed successfully. 🎉 Let me revert the change in `trigger_map`.
```
+ gem install fastlane -v 2.216.0
Successfully installed fastlane-2.216.0
Parsing documentation for fastlane-2.216.0
Installing ri documentation for fastlane-2.216.0
Done installing documentation for fastlane after 6 seconds
1 gem installed
+ fastlane -version
[16:02:32]: fastlane detected a Gemfile in the current directory
[16:02:32]: However, it seems like you didn't use `bundle exec`
[16:02:32]: To launch fastlane faster, please use
[16:02:32]: 
[16:02:32]: $ bundle exec fastlane -version
[16:02:32]: 
[16:02:32]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
fastlane 2.216.0
```